CustomKeystore/src/main/java/com/amazonaws/services/kinesisanalytics/CustomFlinkKafkaUtil.java

```diff
@@ -1,31 +1,22 @@
 package com.amazonaws.services.kinesisanalytics;
 
+import org.apache.commons.io.FileUtils;
 import org.apache.flink.util.FlinkRuntimeException;
 
 import java.io.File;
-import java.io.FileOutputStream;
-import java.io.InputStream;
-import java.io.OutputStream;
+import java.net.URL;
 
 public class CustomFlinkKafkaUtil {
 
     public static void initializeKafkaTruststore() {
```

Comment:
I suggest to add a synchronization to this method to prevent file to be overwritten during high contention (we had this issue with the previous implementation before).